how_to_valid_service.md

```diff
@@ -0,0 +1,405 @@
+# How to Check and Valid Micro Service in the GenAI Example
```

Comment:
Valid -> Validate?  
Suggest to put this doc under `ChatQnA/docker/gaudi`.

Reply:
Validate micro service are common to all examples (and there is no difference between Xeon and Gaudi).
So put it on a common directory for all examples. 

@chensuyue Is there other directory recommened?